src/platform/scene/scene.ts

```diff
@@ -32,6 +33,7 @@ export type Scene = DisplayableDeployment & {
   featureToggles?: FeatureToggles
   worldConfiguration?: WorldConfiguration
   dreamSpaceConfiguration?: DreamSpaceConfiguration
+  portableExperiences?: PortableExperiences
```

Comment:
Hey @nicoecheza. I think both the attribute `portableExperiences` and the type `PortableExperiences` should be singular. Or is it possible for a scene to have more than one? If that's the case, then the type should still be singular and the attribute, while plural, it should be an array.